java/Anton-Kozyk/src/com/zoolatech/lecture3/tasks/_3/Figure.java

```diff
@@ -0,0 +1,9 @@
+package com.zoolatech.lecture3.tasks._3;
+
+public abstract class Figure {
```

Comment:
What is the reason to provide this class? As for me, it looks the same as the interface Shape.